### ISSUE TITLE
Dropdown a11y violations

### ISSUE THREAD

**Issue by hxltrhuxze** [Royal-Navy/design-system#1098]
- [ ] Ensures every form element has a label

**Comment by m7kvqbe1** [comment]
I can't resolve this because we wrap another library - this is a black box.
